s3-other.tf

```diff
@@ -240,16 +240,44 @@ resource "aws_s3_bucket" "apiary_system" {
             "arn:aws:s3:::${local.apiary_system_bucket}/*"
         ]
     },
+%{endif}
+%{if length(var.system_schema_producer_iamroles) > 0}
+    {
+        "Sid": "system schema customer account permissions",
```

Comment:
Why do we need this section specifically for system bucket?

Reply:
the producer iam role does not apply to system buckets and s3 access log, as well as s3 inventory buckets